demo-plugin/src/main/kotlin/org/jetbrains/research/refactoringDemoPlugin/util/PsiUtil.kt

```diff
@@ -33,3 +38,35 @@ fun concatFiltered(
     array2.filterTo(filteredList, condition)
     return filteredList.toTypedArray()
 }
+
+/*
+    Extracts all Java classes from the project.
+ */
+fun extractClasses(project: Project?): MutableList<PsiClass> {
+    val classes: MutableList<PsiClass> = ArrayList()
+
+    ProjectFileIndex.SERVICE.getInstance(project).iterateContent { file: VirtualFile? ->
```

Comment:
Could the iteration code be more concise?
https://github.com/JetBrains-Research/refactoring-workshop-demo/pull/3#discussion_r700453229

Reply:
Done.